tests/composer/composer-symfony3-min.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "propel/propel",
+    "type": "library",
+    "description": "Propel2 is an open-source Object-Relational Mapping (ORM) for PHP 5.5 and up.",
+    "keywords": [
+        "ORM",
+        "persistence",
+        "Active Record"
+    ],
+    "homepage": "http://www.propelorm.org/",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "William Durand",
+            "email": "william.durand1@gmail.com"
+        }
+    ],
+    "require": {
+        "php": ">=7.1",
+        "symfony/yaml": "~3.3.0",
+        "symfony/config": "~3.0.0",
+        "symfony/console": "~3.0.0",
+        "symfony/filesystem": "~3.0.0",
+        "symfony/finder": "~3.0.0",
+        "symfony/translation": "~3.0.0",
+        "symfony/validator": "~3.3.0",
```

Comment:
:red_circle: this is missing in the overall composer.json, there were issues with validator before 3.3.

I have to look deeper into this.